report_templates/host_-_all_installed_packages.erb

```diff
@@ -0,0 +1,24 @@
+<%#
+name: Host - All Installed Packages
+snippet: false
+template_inputs:
+- name: hosts
+  required: false
+  input_type: user
+  advanced: false
+  value_type: plain
+  resource_type: Katello::ActivationKey
+  hidden_value: false
+model: ReportTemplate
+%>
```

Comment:
this needs to be `-%>` otherwise the report would always start with \n if I'm not mistaken, I thought we fixed that in the export, perhaps you used older version of Foreman for the export?

Reply:
Hello @ares I'm doing this report on Sat6.7. In this version, still as above. Also, the line `<%  load_hosts(search: input('hosts')).each_record do |host| -%>` still in the beginning `first line`.

Hope this help.
Waldirio

Reply:
Seems some fixes didn't make it to 6.7, I was able to reproduce the first line issue too